app/styles/ui/_side-by-side-diff.scss

```diff
@@ -241,12 +235,6 @@
       color: var(--diff-hunk-text-color);
       align-items: center;
 
-      div,
-      span {
-        user-select: none;
-        cursor: default;
-      }
-
```

Comment:
From what I've tested, correct me if I'm wrong, but I think we can restore this code in order to prevent selection of "hunk info" rows:
![image](https://github.com/desktop/desktop/assets/1083228/b9dd4c73-ab7c-4f32-a2a0-fde5b4419c63)

After restoring those lines, the text in those rows is not selected but empty lines are:
![image](https://github.com/desktop/desktop/assets/1083228/a341b9e5-5748-4e75-9deb-dc264e529edf)

Reply:
Sorry, I should have called that out. That change was intentional to align with dotcom. I can't think of a strong reason why we shouldn't allow users to copy hunk headers. What's your take?

Reply:
Oh! ~~Well, I don't think they're actually copied despite of being selected~~

Hah! I just tested it and they're copied, as long as it's not due to a Cmd/Control+A or `Select All` action, they are. If you use `Select All` that text is selected but not copied.

I'm more than happy to be consistent with dotcom, but we should probably fix that inconsistency with `Select All`!

Reply:
Ah, good shout, yup we should definitely fix that. I'll get on it Monday!

Reply:
Done in https://github.com/desktop/desktop/pull/17660/commits/914dfd009ce6703b5ab6e0c1e24e2958b17ae42b